src/components/breadcrumb-bar/utils/get-docs-breadcrumbs.ts

```diff
@@ -178,29 +178,47 @@ interface GetDocsBreadcrumbsOpts extends GetPathBreadcrumbsOpts {
   productName: string
   productPath: string
   versions?: VersionSelectItem[]
+  indexOfVersionPathPart?: number
 }
 
 function getDocsBreadcrumbs({
-  basePath,
   baseName,
+  basePath,
+  indexOfVersionPathPart,
```

Comment:
nit: `indexOfVersionPathPart` can be derived from `pathParts` and `version`.

Reply:
💭 ⛏️  if we want to avoid passing this from server, and avoid iterating, maybe we could use [findIndex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex), eg

```ts
const indexOfVersionPathPart = pathParts.findIndex(version)
```